packages/components/psammead-paragraph/package.json

```diff
@@ -22,8 +22,9 @@
   },
   "homepage": "https://github.com/bbc/psammead/blob/latest/packages/components/psammead-paragraph/README.md",
   "dependencies": {
-    "@bbc/gel-foundations": "^0.3.0",
-    "@bbc/psammead-styles": "^0.3.2"
+    "@bbc/gel-foundations": "^1.0.0",
+    "@bbc/psammead-styles": "^0.3.2",
+    "styled-components": "^4.1.2"
```

Comment:
Bringing back styled-components in the wrong dependencies block

Reply:
```suggestion

```

Good spot, `styled-components` isn't needed in the dependencies because it's in the `devDependencies` for storybook and unit tests and it's in the `peerDependencies` for production